lab_1/main.py

```diff
@@ -12,7 +13,13 @@ def tokenize(text: str) -> list:
     e.g. text = 'The weather is sunny, the man is happy.'
     --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
     """
-    pass
+    if type(text) != str:
+        return []
+    text = text.lower()
```

Comment:
Usually it is better to create a separate variable to work with it and leave input as it is.